### ISSUE TITLE
Lesson 6; brownie attaching error with ganache: not getting able to get local RPC client to listen at the local network 

### ISSUE THREAD

**Issue by mattsegura** [smartcontractkit/full-blockchain-solidity-course-py#1080]
<img width="1362" alt="image" src="https://user-images.githubusercontent.com/61678138/154370913-df3a39ca-7ac4-462b-b6c0-7a64d8dd1e7f.png">

- I've tried adding the network manually with `brownie networks add Ethereum ganache-local host=http://127.0.0.1:7545 chainid=5777` but that didn't work
- Everything worked before hand and followed by clicking quick start just as in the video but it just seems like it's not recognizing ganache

**Comment by mattsegura** [comment]
Fixed the issue, needed to go to the Ganache GUI settings -> server and change the port number to 8545 and network id to 1337.

After, deleted the original local network that wasn't working with...

1. `brownie networks delete ganache-local  `

then, added the local ganache network to brownie with...

2. `brownie networks add Ethereum ganache-local host=http://127.0.0.1:8545 chainid=1337`

... once I restarted the Ganache client Brownie attached to the client and deployed the script fine.
<img width="980" alt="image" src="https://user-images.githubusercontent.com/61678138/154378093-1a076df4-66b3-4622-99be-4a5bc54cd592.png">
